packages/amplify-e2e-core/src/init/non-interactive-init.ts

```diff
@@ -28,6 +28,30 @@ export const nonInteractiveInitAttach = async (
   await execa(getCLIPath(), args, { cwd: projRoot });
 };
 
+/**
+ * Executes a non-interactive init to create a new Project
+ */
+export const headlessInit = async (
```

Comment:
could you enrich existing `initHeadless` from `initProjectHelper.ts` instead of creating new function ?